src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -195,11 +196,12 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
 
     /**
      * Returns an unmodifiable view of the list of {@code Record} backed by the internal list of
-     * {@code versionedAddressBook}
+     * {@code versionedAddressBook}. List of {@code Record} returned is sorted by Date, in
```

Comment:
I think `versionedAddressBook` here is wrong, might consider changing the name

Reply:
Agreed, will change it, thanks!